acceptance/.shared/kitchen_acceptance/.kitchen.ec2.yml

```diff
@@ -213,18 +213,7 @@ platforms:
         image-type: machine
       user_data: |
         <powershell>
-        $logfile="C:\\Program Files\\Amazon\\Ec2ConfigService\\Logs\\kitchen-ec2.log"
-        #PS Remoting and & winrm.cmd basic config
-        Enable-PSRemoting -Force -SkipNetworkProfileCheck
-        & winrm.cmd set winrm/config '@{MaxTimeoutms="1800000"}' >> $logfile
-        & winrm.cmd set winrm/config/winrs '@{MaxMemoryPerShellMB="1024"}' >> $logfile
-        & winrm.cmd set winrm/config/winrs '@{MaxShellsPerUser="50"}' >> $logfile
-        #Server settings - support username/password login
-        & winrm.cmd set winrm/config/service/auth '@{Basic="true"}' >> $logfile
-        & winrm.cmd set winrm/config/service '@{AllowUnencrypted="true"}' >> $logfile
-        & winrm.cmd set winrm/config/winrs '@{MaxMemoryPerShellMB="1024"}' >> $logfile
-        #Firewall Config
-        & netsh advfirewall firewall set rule name="Windows Remote Management (HTTP-In)" profile=public protocol=tcp localport=5985 remoteip=localsubnet new remoteip=any  >> $logfile
+        & netsh advfirewall firewall set rule name="Windows Remote Management (HTTP-In)" profile=public protocol=tcp localport=5985 remoteip=localsubnet new remoteip=any
```

Comment:
that was point #3 above but I should have provided more detail:
1. Don't need to log anything because all of stdout/err gets logged to the amazon provided logs on the instance
2. The default winrm settings on win2012r2 for things like memory and timeouts are much higher than previously in 2k8r2 and sufficint for our purposes
3. Latest TK and winrm use negotiate auth so basic auth without encryption is not necessary or desired
4. winrm is on by default on 2012r2 so there is no need to enable it